wherehows-data-model/DDL/ETL_DDL/metric_metadata.sql

```diff
@@ -34,18 +34,52 @@ CREATE TABLE dict_business_metric  (
   `metric_display_factor_sym`	VARCHAR(20) COMMENT '%, (K), (M), (B), (GB), (TB), (PB)' NULL,
   `metric_good_direction`	VARCHAR(20) COMMENT 'UP, DOWN, ZERO, FLAT' NULL,
   `metric_formula`           	TEXT COMMENT 'Expression, Code Snippet or Calculation Logic' NULL,
-  `dimensions`			VARCHAR(500) CHAR SET latin1 NULL,
+  `dimensions`			VARCHAR(800) CHAR SET latin1 NULL,
   `owners`                 	VARCHAR(300) NULL,
   `tags`			VARCHAR(300) NULL,
   `urn`                         VARCHAR(300) CHAR SET latin1 NOT NULL,
   `metric_url`			VARCHAR(300) CHAR SET latin1 NULL,
   `wiki_url`              	VARCHAR(300) CHAR SET latin1 NULL,
   `scm_url`               	VARCHAR(300) CHAR SET latin1 NULL,
+  `wh_etl_exec_id`              BIGINT COMMENT 'wherehows etl execution id that modified this record',
   PRIMARY KEY(metric_id),
+  UNIQUE KEY `uq_dataset_urn` (`urn`),
   KEY `idx_dict_business_metric__urn` (`urn`) USING BTREE,
```

Comment:
this urn is for loading "on duplicate key update...", if metric urn is the same, then update is performed.